EventFilter/GEMRawToDigi/python/GEMSQLiteCabling_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Can this file be removed once the relevant condition is in the database?

Reply:
yes